helm-charts/helm-api/ibm_cloud_pak/manifest.yaml

```diff
@@ -1,111 +1,142 @@
+    platform:
+      os: linux
+      arch: ppc64le
+  - repository: icp-audit-service@sha256:24a220374c39652f8b031637d852995bfaec5be2601f9e4c41eb1925531f466d
+    pull-repository: quay.io/opencloudio/icp-audit-service@sha256:24a220374c39652f8b031637d852995bfaec5be2601f9e4c41eb1925531f466d
+    platform:
+      os: linux
+      arch: s390x
 
-- image: icp-audit-service:3.3.1
+- image: icp-helm-rudder:3.3.0
   references:	
-  - repository : icp-audit-service-amd64:3.3.1
-    pull-repository: hyc-cloud-private-edge-docker-local.artifactory.swg-devops.com/ibmcom-amd64/icp-audit-service:3.3.1
-    pull-authorization:
-      username:
-        env: ARTIFACTORY_USERNAME
-      password:
-        env: ARTIFACTORY_APIKEY
-    platform:
-      os: linux
-      architecture: amd64
-  - repository : icp-audit-service-ppc64le:3.3.1
-    pull-repository: hyc-cloud-private-edge-docker-local.artifactory.swg-devops.com/ibmcom-ppc64le/icp-audit-service:3.3.1
-    pull-authorization:
-      username:
-        env: ARTIFACTORY_USERNAME
-      password:
-        env: ARTIFACTORY_APIKEY
-    platform:
-      os: linux
-      architecture: ppc64le
-  - repository : icp-audit-service-s390x:3.3.1
-    pull-repository: hyc-cloud-private-edge-docker-local.artifactory.swg-devops.com/ibmcom-s390x/icp-audit-service:3.3.1
-    pull-authorization:
-      username:
-        env: ARTIFACTORY_USERNAME
-      password:
-        env: ARTIFACTORY_APIKEY
-    platform:
-      os: linux
-      architecture: s390x
-
+  - repository : icp-helm-rudder:3.3.0-amd64
+    pull-repository: quay.io/opencloudio/icp-helm-rudder:3.3.0-amd64
+    platform:
+      os: linux
+      arch: amd64
+  - repository : icp-helm-rudder:3.3.0-ppc64le
+    pull-repository: quay.io/opencloudio/icp-helm-rudder:3.3.0-ppc64le
+    platform:
+      os: linux
+      arch: ppc64le
+  - repository : icp-helm-rudder:3.3.0-s390x
+    pull-repository: quay.io/opencloudio/icp-helm-rudder:3.3.0-s390x
+    platform:
+      os: linux
+      arch: s390x
+- image: icp-helm-rudder@sha256:5eba2b4052eb80a7ed1e0a91f3e0bcd1e4ae8412e604cfd4dc990daf2e9da9c7
+  references:	
+  - repository : icp-helm-rudder@sha256:5eba2b4052eb80a7ed1e0a91f3e0bcd1e4ae8412e604cfd4dc990daf2e9da9c7
+    pull-repository: quay.io/opencloudio/icp-helm-rudder@sha256:5eba2b4052eb80a7ed1e0a91f3e0bcd1e4ae8412e604cfd4dc990daf2e9da9c7
+    platform:
+      os: linux
+      arch: amd64
+  - repository : icp-helm-rudder@sha256:5eba2b4052eb80a7ed1e0a91f3e0bcd1e4ae8412e604cfd4dc990daf2e9da9c7
+    pull-repository: quay.io/opencloudio/icp-helm-rudder@sha256:5eba2b4052eb80a7ed1e0a91f3e0bcd1e4ae8412e604cfd4dc990daf2e9da9c7
+    platform:
+      os: linux
+      arch: ppc64le
+  - repository : icp-helm-rudder@sha256:5eba2b4052eb80a7ed1e0a91f3e0bcd1e4ae8412e604cfd4dc990daf2e9da9c7
+    pull-repository: quay.io/opencloudio/icp-helm-rudder@sha256:5eba2b4052eb80a7ed1e0a91f3e0bcd1e4ae8412e604cfd4dc990daf2e9da9c7
```

Comment:
icp-helm-rudder is at 3.3.1 and digest is `437c980277a700f7f53db1ac88ae1d10488033b52422bf17654a82717c6f22af` now